app/views/batches/_request.html.erb

```diff
@@ -1,75 +0,0 @@
-
```

Comment:
Part of the detailed worksheet. Used `request.value_for(key)` but without request information types. I did start to update it, but then found the page hadn't worked for some time, and didn't make any sense in the contexts in which it was used.